### ISSUE TITLE
Multiple Subscription changes broke Stripe::Customer#update_subscription support

### ISSUE THREAD

**Issue by thewoolleyman** [stripe-ruby-mock/stripe-ruby-mock#70]
Hello,

The following commit removed support for the '/v1/customers/{CUSTOMER_ID}/subscription' endpoint:

https://github.com/rebelidealist/stripe-ruby-mock/commit/1fd490818e2ad575f45ab9a863e8d749521a63d9#diff-fe9a4c81b76150e0bf2da5cb7d208802L8

However, this endpoint is still supported by the API:

https://stripe.com/docs/api#intro
https://github.com/stripe/stripe-ruby/blob/master/lib/stripe/customer.rb#L38

Also, I cannot see, with the current (multiple) Subscriptions support in v1.10.1.2, how I am supposed to mock update of a subscription.  The Stripe api docs suggest using 'customer#subscriptions', but the current mocking of the subscriptions array in the customer response object is incomplete - it does not contain a list of subscription objects (or subscription IDs, which are necessary to update them).

So, in other words, the old approach was removed, and the new approach does not work as far as I can see.  Do you have the v1.10.1.2 version working to mock any production code which retrieves or updates subscriptions?  If so, what exact calls are you using to do it?

Thanks,
-- Chad
